packages/mds-db/index.ts

```diff
@@ -1119,14 +1119,20 @@ async function readSingleGeography(geography_id: UUID): Promise<Geography> {
   }
 }
 
-async function readGeographies(params?: { get_read_only?: boolean }): Promise<Geography[]> {
+async function readGeographies(params?: { get_read_only?: boolean; summary?: boolean }): Promise<Geography[]> {
   // use params to filter
   // query on ids
   // return geographies
   try {
     const client = await getReadOnlyClient()
 
-    let sql = `select * from ${schema.TABLE.geographies}`
+    const cols =
+      params && params.summary
```

Comment:
I am *SO* looking forward to TS 3.7 `params?.summary`

Reply:
See above... `Partial` with initializer and defaults avoids this.